source/reference/api/start.txt

```diff
@@ -406,4 +439,21 @@ To check whether it is safe to rename collections, call the
   Renaming the collection from this point on does not block
   its sharding on the destination cluster, even in the event  of a crash.
 
+.. _c2c-required-indexes:
+
+Required Indexes
+~~~~~~~~~~~~~~~~
+
+When you call ``/start`` with the ``buildIndexes`` option set to ``never``,
+``mongosync`` skips building unnecessary indexes.  
+
+Indexes that are always built include:
+
+* ``mongosync`` automatically builds an index for the ``_id`` field on every
+  collection it syncs.
 
+* Starting in MongoDB 6.0, ``mongosync`` builds indexes for a ``dummy``
+  shard key index on sharded collections, which is removed after commit.
```

Comment:
```suggestion
* Starting in MongoDB 6.0, ``mongosync`` builds dummy indexes that support the shard key for each sharded collection, which are removed after commit.
```

Reply:
This is mostly a suggestion, I just think the current wording is still a bit confusing.